docs/guide.rst

```diff
@@ -87,6 +87,21 @@ Here's how you can deploy new contracts:
     :dedent: 4
 
 
+Deploying new contracts with Universal Deployer Contract (UDC)
+--------------------------------------------------------------
+
+The best way of deploying contracts on Starknet is to use UDC. starknet.py assumes you use an implementation compatible
+with `OpenZeppelin's UDC implementation <https://github.com/OpenZeppelin/cairo-contracts/blob/main/src/openzeppelin/utils/presets/UniversalDeployer.cairo>`_.
```

Comment:
I think providing the interface would be more appropriate here, since it should be 1 function only, right? And if the contract shares that method, it'll be compatible. It doesn't even have to be a UDC